tests/console/salt.pm

```diff
@@ -33,9 +33,8 @@ EOF
     assert_script_run('salt-run state.event tagmatch="salt/auth" quiet=True count=1', timeout => 300);
     assert_script_run("salt-key --accept-all -y");
     # try to ping the minion. If it does not respond on the first try the ping
-    # might have gone lost so try more often
+    # might have gone lost so try more often. Also see bsc#1069711
     unless (script_run 'salt \'*\' test.ping') {
```

Comment:
Having this `salt` command in one place would be nice if you want to boy-scout ;)